### PR TITLE
- gcemetadata

### PR DESCRIPTION
**Pull request by rjschwei**
+ pep8 compliance
  + Modify function, method, and variable names

### CODE REVIEW

File: gcemetadata/gcemetadata [hunk 1]
```diff
@@ -25,40 +25,44 @@ import gcemetadata.gcemetautils as gcemetautils
...
     for opt in options.keys():
         if type(options[opt]) == dict:
             if opt in opts_with_id_targets:
                 # Figure out the ID
-                devIDs = options[opt].keys()
-                devIDs.sort()
-                for devID in devIDs:
+                option_ids = options[opt].keys()
+                option_ids.sort()
+                for opt_id in option_ids:
```
Comment (bear454) [inline gcemetadata/gcemetadata]: I'd prefer the singular match the plural; don't see any good reason to shorten it. Consider:

```
for option_id in option_ids:
```

File: gcemetadata/lib/gcemetadata/gcemetadata.py [hunk 2]
```diff
@@ -21,47 +21,44 @@
 
 from gcemetaExceptions import *
```
Comment (smarlowucf) [inline gcemetadata/lib/gcemetadata/gcemetadata.py:23]: Ditto

File: gcemetadata/gcemetadata [hunk 3]
```diff
@@ -25,40 +25,44 @@ import gcemetadata.gcemetautils as gcemetautils
 
 from gcemetadata.gcemetaExceptions import *
```
Comment (smarlowucf) [inline gcemetadata/gcemetadata:26]: Wildcard imports should be avoided.

File: gcemetadata/lib/gcemetadata/gcemetautils.py [hunk 4]
```diff
@@ -26,43 +26,46 @@
 
 from gcemetaExceptions import *
```
Comment (smarlowucf) [inline gcemetadata/lib/gcemetadata/gcemetautils.py:27]: Ditto

**Review by smarlowucf — CHANGES_REQUESTED** [review]
Looks good, a few other things:

Pep8:
- setup.py:40 remove whitepspace before colon
- gcemetaExceptions.py:18 expected 2 blank lines before class
- gcemetadata.py:69 `e` is never used

Syntax:
- gcemetadata.py:70 time is undefined
- gcemetautils.py:75 s/filePath/path
- setup.py:24-25 sys is undefined

File: gcemetadata/gcemetadata (hunk 1 above)
Comment (rjschwei) [reply gcemetadata/gcemetadata]: Done

File: gcemetadata/lib/gcemetadata/gcemetautils.py (hunk 4 above)
Comment (rjschwei) [reply gcemetadata/lib/gcemetadata/gcemetautils.py:27]: Sorry, in context I don't see where "import *" does any harm in potentially confusing name spaces, thus not really seeing a need for a change.

Everything else is done

Comment (smarlowucf) [reply gcemetadata/lib/gcemetadata/gcemetautils.py:27]: Agreed, but it is mentioned as part of [pep8](https://www.python.org/dev/peps/pep-0008/#imports) even if `pep8` doesn't raise a warning. The `flake8` wrapper however does raise exception for all instances of wildcard usage + usage of imports that aren't explicitly imported.

E.g.
`F405 'GCEMetadataException' may be undefined, or defined from star imports: gcemetaExceptions`
`F403 'from gcemetaExceptions import *' used; unable to detect undefined names`

**Review by smarlowucf — APPROVED** [review]

**Review by bear454 — APPROVED** [review]

**Pull request merged**